### PR TITLE
1.2 final

### PR DESCRIPTION
**Pull request by TechniumUnlimited**
This is the final PR for merging to master of changes implemented for 1.2.0 of Ignition coin wallets, changes include:

PoW End at block 521352 ~Nov 15th
PoS/MN Reward Ratio Change from 50/50 to 45/55 % also at block 521352 ~Nov 15th
A Bi-Yearly Max Blocksize Increase System for On-Chain Scaling that starts in 4 years (6th year from genesis block approx.)
Some Minor Bug Fixes and Improvements

### CODE REVIEW

File: src/main.h [hunk 1]
```diff
@@ -57,7 +57,7 @@ static const unsigned int MAX_BLOCK_SIGOPS = MAX_BLOCK_SIZE/50;
 /** Maxiumum number of signature check operations in an IsStandard() P2SH script */
 static const unsigned int MAX_P2SH_SIGOPS = 15;
 /** The maximum number of sigops we're willing to relay/mine in a single tx */
-static const unsigned int MAX_TX_SIGOPS = MAX_BLOCK_SIGOPS/5;
+static const unsigned int MAX_TX_SIGOPS = MAX_BLOCK_SIGOPS/5;;
```
Comment (fopat) [inline src/main.h]: Unnecessary period added here :)

File: src/main.h [hunk 2]
```diff
@@ -708,6 +722,10 @@ class CBlock
         if (nBestHeight == 0) {
             return CURRENT_BLOCK_VERSION_1;
         }
+        if(nBestHeight >= GetForkHeightTwo()-5)
```
Comment (fopat) [inline src/main.h]: -5 is not necessary, all the other changes are at the same height than fork two

Comment (TechniumUnlimited) [reply src/main.h]: Thanks

File: src/main.h (hunk 1 above)
Comment (TechniumUnlimited) [reply src/main.h]: lol

File: src/main.h (hunk 2 above)
Comment (TechniumUnlimited) [reply src/main.h]: oh, wait, I thought you might have changed it, didn't realize these were just comments on past changes xD

**Comment by TechniumUnlimited** [comment]
Okay, issues resolved that @fopat pointed out

**Pull request merged**